api/envoy/config/filter/http/aws_lambda/v2alpha/aws_lambda.proto

```diff
@@ -0,0 +1,36 @@
+syntax = "proto3";
+
+package envoy.config.filter.http.aws_lambda.v2alpha;
+
+import "udpa/annotations/status.proto";
+
+import "udpa/annotations/migrate.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.config.filter.http.aws_lambda.v2alpha";
+option java_outer_classname = "AwsLambdaProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_migrate).move_to_package =
+    "envoy.extensions.filters.http.aws_lambda.v3";
+option (udpa.annotations.file_status).work_in_progress = true;
+
+// [#protodoc-title: AwsLambda]
+// AwsLambda :ref:`configuration overview <config_http_filters_aws_lambda>`.
+// [#extension: envoy.filters.http.aws_lambda]
+
+// AWS Lambda filter config
+message Config {
+  // The ARN of the AWS Lambda to invoke when the filter is engaged
+  // Must be in the following format:
+  // arn:aws:lambda:<region>:<account-number>:function:<function-name>
+  string arn = 1 [(validate.rules).string = {min_len: 27}];
```

Comment:
where did `27` come from? I can't figure out the math here..

Reply:
- 6 colons
- `arn` is 3 letters (never changes)
- `lambda` is 6 letters (can change but not in this config)
-  `function` is 8 letters (also can't be anything else in this context)
- at least one character in (partition, region, account, and function name)

That adds up to 27. Open to suggestions on how to document that.

Reply:
Talked offline.. changing this to 1 since it's not obvious and hard to document.